<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAplikasi">
    <property name="title">
     <string>Aplikasi</string>
    </property>
    <addaction name="actionSuhu"/>
    <addaction name="actionPrisma"/>
    <addaction name="actionGravitasi"/>
    <addaction name="actionBMI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAplikasi"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSuhu">
   <property name="text">
    <string>Konversi Suhu</string>
   </property>
  </action>
  <action name="actionPrisma">
   <property name="text">
    <string>Prisma</string>
   </property>
  </action>
  <action name="actionGravitasi">
   <property name="text">
    <string>Gravitasi</string>
   </property>
  </action>
  <action name="actionBMI">
   <property name="text">
    <string>Body Mass Index</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
